rice-framework/krad-app-framework/src/main/java/org/kuali/rice/krad/UserSession.java

```diff
@@ -216,7 +216,7 @@ public String addObjectWithGeneratedKey(Serializable object, String keyPrefix) {
      *
      * @param object
      */
-    public String addObjectWithGeneratedKey(Object object) {
+    public String addObjectWithGeneratedKey(Serializable object) {

```

Comment:
This, of course, changes the api and would break compatibility. Given though that this is contrained to Rice standalone server, I think that's fine. And if you are sticking stuff in a session it should be serializable anyway.